.travis.yml

```diff
@@ -38,7 +41,8 @@ env:
     - secure: "EvZbFFwD/XggLEy7H6yY/6OQrFbFbsRjRPrT1Hb8/TDGE8VlwC7us7xFSVIkLhj9K//VvEQucviP/bGg2EIziRng8oSz4x+X6CM0hra3PdmogZzMtBUFoY7olY5c/opo967WO8QZ+2yIArmvB1RbiQ6eh82l13qaI+R8wfIjxT+JpAjMHUGnRmqx1sUuM7ZQYQKlpt0Q9sFaPh8U2FEgspi9auvHp7WOCaJR8uGrHImfvf/c7lzxWjcImqaT357XJnxwPbV418J3p/cOpgHvjJD2jTA8yY0qo62rsemqYyhzCwaTU+Q0Sz5ykUW80a49tW3iGRVo1K0grzL9TcV3HAVNb419YBgKm2mko4hG/JpDyphydVRLvJLv+jv6kb8Iyx5N3h5m/n782EJIoitNxObnuqM9L+RqO5XcqCNBqfD38NSShvp6WKPUa8JVMU+d+yJ5rfFLcJxIWRxSGvoSi7uVuQ9ceOJRHhuV3O6p67KhDfd7/79xtxERxXU/9gwIliSURXU4r/uU7agr/jlKAP7P/6btQrfuxiEhFNr6Yc1YmxnF3ul/uzQyOZIa6nN1mzGrCtO5bLXMo9b66mPOaW4OXjIGELWD99Q1xIolj92w7cKRp5iKm5fnBuvbSnsQ1xq3GSNfQWp6FzxHhvxVL4C42tfth61kpHuxwbY7U9o="
```

Comment:
@asvetlov do you remember what's in this global var? We use the deployment provider directly... Do we still need this?

Reply:
I don't. Let's check deployment (and fix if necessary) in another PR.

Reply:
Looks like it was an attempt to figure out adding creds: https://github.com/aio-libs/yarl/commit/f825a5d7068e52346615e87b9f5c1753b38d73d3